app/src/main/java/com/example/coronadiagnosticapp/ui/activities/OxymeterActivity.java

```diff
@@ -435,29 +212,55 @@ public void onTick(long millisUntilFinished) {
                 } else {
                     tv_time.setText(seconds + " seconds");
```

Comment:
It's a good idea to display this but:
1. I couldn't find a reference to this in the actual layout
2. It shouldn't use hardcoded seconds (needs to be taken from string resources to later support translation